### PR TITLE
Node 180, 219 it activation protocol

### PR DESCRIPTION
**Pull request by ebceu4**

### CODE REVIEW

File: src/it/resources/template.conf [hunk 1]
```diff
@@ -28,9 +28,7 @@ waves {
         allow-leased-balance-transfer-until = 0
         enable-microblocks-after-height = 0
         block-version-3-after = 0
-        pre-activated-features = {
-          2 = 0
-        }
+        pre-activated-features = {}
```
Comment (ismagin) [inline src/it/resources/template.conf]: NG should be enabled in it-tests, please restore

File: src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.wavesplatform.it
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import org.scalatest.{BeforeAndAfterAll, FreeSpec, Matchers}
+
+import scala.concurrent.Await
+
+class ActivationTestSuite extends FreeSpec with Matchers with BeforeAndAfterAll with IntegrationNodesInitializationAndStopping {
```
Comment (ismagin) [inline src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala]: What is the point of this test?

File: src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  override val docker = new Docker()
+  override val nodes = Docker.NodeConfigs.getConfigList("nodes").asScala.take(2).map(docker.startNode)
+
+  "api consuming example" in {
+    Await.result(nodes.head.waitForHeight(5), 5.minute)
```
Comment (ismagin) [inline src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala]: Use `Future.traverse` to await for all nodes to get to some state

File: src/main/scala/com/wavesplatform/Application.scala [hunk 4]
```diff
@@ -90,7 +91,8 @@ class Application(val actorSystem: ActorSystem, val settings: WavesSettings) ext
       DebugApiRoute(settings.restAPISettings, wallet, stateReader, history, peerDatabase, establishedConnections, blockchainUpdater, allChannels, utxStorage, blockchainDebugInfo, miner),
       WavesApiRoute(settings.restAPISettings, wallet, utxStorage, allChannels, time),
       AssetsApiRoute(settings.restAPISettings, wallet, utxStorage, allChannels, stateReader, time),
-      NodeApiRoute(settings.restAPISettings, () => this.shutdown()),
+      NodeApiRoute(settings.restAPISettings,() => this.shutdown()),
+      ActivationApiRoute(settings.restAPISettings, settings.blockchainSettings.functionalitySettings, settings.featuresSettings, history, featureProvider),
```
Comment (ismagin) [inline src/main/scala/com/wavesplatform/Application.scala]: might as well provide the whole `settings`

File: waves-nopeers.conf [hunk 5]
```diff
@@ -0,0 +1,149 @@
+# Waves node settins
```
Comment (ismagin) [inline waves-nopeers.conf]: There must be a better way to broadcast override only `known-peers`. What is this used for?

File: src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala (hunk 2 above)
Comment (ebceu4) [reply src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala]: api consuming example

File: src/main/scala/com/wavesplatform/Application.scala (hunk 4 above)
Comment (ebceu4) [reply src/main/scala/com/wavesplatform/Application.scala]: might not ;)

File: src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala (hunk 3 above)
Comment (ebceu4) [reply src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala]: If you want to test particular node, what's the point to wait for others?

**Pull request merged**